doc/changelog/07-commands-and-options/10185-instance-no-bang.rst

```diff
@@ -0,0 +1,2 @@
+- Removed undocumented ``Instance : !type`` syntax
+  by Gaëtan Gilbert
```

Comment:
@Zimmi48 this is my first change entry in the new system, is that how it's supposed to look?

Reply:
What about:

```rst
- Remove undocumented :n:`Instance : !@type` syntax
  (`#10185 <https://github.com/coq/coq/pull/10185>`_, by Gaëtan Gilbert).
```

?

Reply:
The link to the PR isn't automatically generated?

Reply:
No. For now, this is all very basic. We can try to move to something fancier later on.

Reply:
Changed.